domain/user/service/service.go

```diff
@@ -49,6 +49,11 @@ type State interface {
 	// returned.
 	GetUserByName(context.Context, string) (user.User, error)
 
+	// GetAllUsers will retrieve all users from the database where the user is
+	// active and has not been removed. If no users exist an empty slice will be
+	// returned.
+	GetAllUsers(context.Context) ([]user.User, error)
```

Comment:
You need to expose disabled users based on the call site https://github.com/juju/juju/blob/main/apiserver/facades/client/usermanager/usermanager.go#L258

Or we just expose all the users including disabled user?